ios/FluentUI/Obscurable/DimmingView.swift

```diff
@@ -18,13 +18,34 @@ public enum DimmingViewType: Int {
 
 @objc(MSFDimmingView)
 open class DimmingView: UIView {
-    public struct Constants {
-        public static let blackAlpha: CGFloat = 0.4
-        public static let whiteAlpha: CGFloat = 0.5
+    private struct Constants {
+        static let blackAlpha: CGFloat = 0.4
+        static let whiteAlpha: CGFloat = 0.5
     }
 
     private var type: DimmingViewType
 
+    /// override default black dimmed color
+    public var dimmedBlackColor: UIColor? {
+        didSet {
+            setBackground(type: type)
+        }
+    }
```

Comment:
Likely mostly harmless, but it seems like this was part of the change to add tokens to the drawer, #476 , which hasn't made its way in to main yet. So, I don't think any of the optional colors added here would actually be used in the DrawerPresentationController.

Reply:
Hmm I'll take a look.